### PR TITLE
Add support for spaces in title/description

### LINKED ISSUE #10: Milestone not created, spaces messing things up?
I've been trying to use this action but I've encountered an interesting issue. I believe that the space in my milestone name is messing it up.

## Info

* Milestone name example: Pods 3.0.11.4 (just a test one as I've been testing this action)
* My GitHub action: https://github.com/pods-framework/pods/actions/workflows/prepare-release-branch.yml
* Error in latest run: https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:18

## Action usage

```
      - name: Create Milestone
        id: create-milestone
        uses: WyriHaximus/github-action-create-milestone@v1.2.0
        continue-on-error: true
        with:
          title: '${{ env.MILESTONE_PREFIX }}${{ env.PLUGIN_VERSION }}'
        env:
          GITHUB_TOKEN: ${{ secrets.GH_BOT_PUSH }}
```

## Action output

```
Run WyriHaximus/github-action-create-milestone@v1.2.0
  with:
    title: Pods 3.0.11.4
  env:
    PR_PREFIX: Pods 
    PR_LABELS: Type: Release
    MILESTONE_PREFIX: Pods 
    BRANCH_PREFIX: release/
    WPORG_PLUGIN_FILE: init.php
    WPORG_PLUGIN_VERSION_CONSTANT_NAME: PODS_VERSION
    PLUGIN_VERSION: 3.0.11.4
    GITHUB_TOKEN: ***
/usr/bin/docker run --name a[2](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:2)86f1d191124f[3](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:3)[4](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:4)[5](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:5)e3a02c5e55[6](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:6)6b5049[7](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:7)_ce[8](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:8)757 --label 063a28 --workdir /github/workspace --rm -e "PR_PREFIX" -e "PR_LABELS" -e "MILESTONE_PREFIX" -e "BRANCH_PREFIX" -e "WPORG_PLUGIN_FILE" -e "WPORG_PLUGIN_VERSION_CONSTANT_NAME" -e "PLUGIN_VERSION" -e "GITHUB_TOKEN" -e "INPUT_TITLE" -e "INPUT_DESCRIPTION" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_RESULTS_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/pods/pods":"/github/workspace" 063a28:6f1d1[9](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:9)[11](https://github.com/pods-framework/pods/actions/runs/7190822982/job/19584555536#step:10:11)24f345e3a02c5e5566b50497
♫♪.ılılıll|̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅|milestone|̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅|llılılı.♫♪
--------{---(@Pods@}}>-----
--------{---(@3.0.11.4@}}>-----
jq: error: Invalid numeric literal at EOF at line 1, column 8 (while parsing '3.0.11.4') at <top-level>, line 1:
3.0.11.4
jq: 1 compile error
```

### PR DESCRIPTION
**Pull request by sc0ttkclark**
Fixes #10

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -13,12 +13,12 @@ if [ $(echo ${INPUT_TITLE} | wc -c) -eq 1 ] ; then
 fi
 
 printf "\u001b[30;40m♫♪.ılılıll|̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅|%s|̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅|llılılı.♫♪\u001b[0m\n" "milestone"
-printf "\u001b[30;40m--------{---(@%s@}}>-----\u001b[0m\n" ${INPUT_TITLE}
+printf "\u001b[30;40m--------{---(@%s@}}>-----\u001b[0m\n" "${INPUT_TITLE}"
 
 if [ $(echo ${INPUT_DESCRIPTION} | wc -c) -gt 1 ] ; then
```
Comment (sc0ttkclark) [inline entrypoint.sh]: Unsure if this also needs quoting

```suggestion
if [ $(echo "${INPUT_DESCRIPTION}" | wc -c) -gt 1 ] ; then
```

Comment (sc0ttkclark) [reply entrypoint.sh]: Same with the input title empty check on line 10

Comment (WyriHaximus) [reply entrypoint.sh]: @sc0ttkclark Go for it, think it's wiser to do anyway. Once this is merged and released I'll figure out a way to test the API calls to GitHub to unit test this thing to be sure it fully works as intended.

Comment (sc0ttkclark) [reply entrypoint.sh]: I'll get those done today for you. Thanks for being so receptive to the PR 😄 

I have really enjoyed being able to have an action for this to automate more processes.

Comment (sc0ttkclark) [reply entrypoint.sh]: Adjustments done

**Review by WyriHaximus — APPROVED** [review]
Thanks for the updates :+1:

**Comment by sc0ttkclark** [comment]
I'll test this and confirm on my pods-framework/pods repo that it's working for us now. Thanks!

File: entrypoint.sh (hunk 1 above)
Comment (WyriHaximus) [reply entrypoint.sh]: > I'll get those done today for you. Thanks for being so receptive to the PR 😄

Sweet thanks, will start working on the testing updates later tonight or tomorrow before relating these changes.

> I have really enjoyed being able to have an action for this to automate more processes.

Let me know if you want to bounce some ideas off me for that. Got my entire release process down to the click of closing a milestone. Having a milestone added to every PR is key to that, this is the or one of the first Actions I've created.

Comment (sc0ttkclark) [reply entrypoint.sh]: Oh snap I didn't even consider that. Setting up a new release/milestone on close would be pretty cool too. I'll explore that.

**Comment by sc0ttkclark** [comment]
Just confirming this is good now

**Comment by WyriHaximus** [comment]
@sc0ttkclark sweet! Thanks for the update 👍

**Pull request merged**